src/open_inwoner/configurations/bootstrap/siteconfig.py

```diff
@@ -0,0 +1,105 @@
+from django.conf import settings
+
+from django_setup_configuration.configuration import BaseConfigurationStep
+
+from open_inwoner.configurations.models import SiteConfiguration
+
+
+class SiteConfigurationStep(BaseConfigurationStep):
+    """
+    Set up general configuration ("Algemene configuratie")
+    """
+
+    verbose_name = "Site configuration"
+    required_settings = [
+        "SITE_NAME",
+        "SITE_SECONDARY_COLOR",
+        "SITE_ACCENT_COLOR",
+    ]
+
+    def is_configured(self):
+        return all((getattr(settings, required) for required in self.required_settings))
```

Comment:
This should check if the required fields are set on the `SiteConfiguration` singleton, otherwise `is_configured` will always be true (because the settings have to be defined), causing the `configure` method to be skipped